oioswift/proxy/controllers/obj.py

```diff
@@ -383,6 +383,10 @@ def _store_object(self, req, data_source, headers):
             raise HTTPClientDisconnect(request=req)
         except ServiceBusy:
             raise
+        except exceptions.ClientException as err:
+            # 481 = CODE_POLICY_NOT_SATISFIABLE
+            if err.status == 481:
+                raise ServiceBusy
```

Comment:
Why `ServiceBusy` for a policy error ?

Reply:
I don't know which error can be used ?

Reply:
This error often happens when oiosds is on heavy load,
 but it's true that a policy error not due to a heavy load can happen.